docker/env/hydra.sh

```diff
@@ -9,6 +9,7 @@ CENTOS_PREREQS_FILE=install-prereqs.sh
 WORK_DIR=/sct
 HOST_NAME=SCT-CONTAINER
 export SCT_TEST_ID=${SCT_TEST_ID:-$(uuidgen)}
+export SCT_GIT_USER_EMAIL=$(git config --get user.email)
```

Comment:
using SCT_ would break sct_config since it's not documented option

Reply:
fixed